gulpfile.js

```diff
@@ -0,0 +1,48 @@
+var gulp = require('gulp');
+var path = require('path');
+var browserify = require('browserify');
+var babel = require('gulp-babel');
+var derequire = require('gulp-derequire');
+var insert = require('gulp-insert');
+var rename = require('gulp-rename');
+var source = require('vinyl-source-stream');
+var uglify = require('gulp-uglify');
+
+var BUILD = 'browser';
+var DIST = './';
+var VERSION = require('./package.json').version;
+
+var FULL_HEADER = (
+	'/**\n' +
+	' * SocketCluster JavaScript client v' + VERSION + '\n' +
+	' */\n');
+
+gulp.task('browserify', function () {
+	var stream = browserify({
+			builtins: ['_process', 'events', 'buffer', 'querystring'],
```

Comment:
Can you use 2-space indentation (for consistency with the existing code base). 2-spaces is most the most common Node.js indentation style.